styles/main.css

```diff
@@ -665,6 +809,21 @@ form .cta-button:hover {
     flex-direction: row-reverse;
   }
 
+  .popup-bg {
+    margin-left: -8rem;
+    margin-top: -11rem;
+  }
+
+  .popup-container {
+    width: 90%;
+    justify-self: center;
+    margin-top: -11rem;
+  }
+
+  .popup-card {
+    justify-self: center;
+  }
```

Comment:
- [ ] Let's make sure our image in the pop-up is the same size in desktop view to design to make your project more appealing and get read of `negative space`

Your implementation | Figma Design
:------:|:-----:
![image](https://user-images.githubusercontent.com/87689547/156900254-1bf62316-61a1-4603-96b1-366eedcc5ac0.png)| ![image](https://user-images.githubusercontent.com/87689547/156900262-08c5cb91-007e-45b5-b588-5219a5bc774f.png)

Reply:
Done, thank you!